### PR TITLE
Migrate MySQL dialect to its own repo

### LINKED ISSUE #431: Move `MySql` dialect to its own repository
Move `MySql` dialect to its own repository

### PR DESCRIPTION
**Pull request by chiaradiamarcelo**
Closes #431.

### CODE REVIEW

File: doc/dialects/mysql.md [hunk 1]
```diff
@@ -1,129 +0,0 @@
-# MySQL SQL Dialect
-
-[MySQL](https://www.mysql.com/) is an open-source relational database management system.
```
Comment (jakobbraun) [inline doc/dialects/mysql.md:3]: Add a link to the new dialect instead. Some websites could still point on that page

File: doc/dialects/db2.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ The SQL statement below creates the adapter script, defines the Java class that
 ```sql
 CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
   %scriptclass com.exasol.adapter.RequestDispatcher;
-  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-bundle-4.0.5.jar;
+  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-bundle-5.0.0.jar;
```
Comment (jakobbraun) [inline doc/dialects/db2.md:59]: Is this really a new major version? In that case shouldn't we have more major versions?

**Review by jakobbraun — APPROVED** [review]

File: doc/dialects/db2.md (hunk 2 above)
Comment (chiaradiamarcelo) [reply doc/dialects/db2.md:59]: well, as we change the interfaces (in this case we remove a whole bunch of it), I would say yes.

And second question, probably yes. @redcatbear what do you think?

Comment (redcatbear) [reply doc/dialects/db2.md:59]: If and only if you make the interface incompatible, increase the major version.

**Pull request merged**